.github/workflows/lint_and_unit_test.yml

```diff
@@ -0,0 +1,47 @@
+name: Android Lint & Unit Test flow
```

Comment:
While we still don't have conventions for workflow naming, I would like to recommend to name workflow using verbs e.g. `test.yml`, `lint.yml`, `distribute.yml`... And second rule, adding the environment if needed e.g. `distribute-staging.yml`, `distribute-production.yml`...

The name attribute can be anything but keeping it short would be great as it can be referenced in other worflows as in:

```
on:
  workflow_run:
    workflows:
      - Test <- this is actually the name attribute 😅 So imagine now if it has to be Android Lint & Unit Test flow
    branches:
      - master
      - development
```

So maybe the names can be just `name: Test` and let the steps describe what it does 💭

Reply:
@olivierobert for the top `name` attribute I agree we can make it short. 
At first, I separated into multiple `steps` but apparently, we had to run a few duplication setups in each of the steps, so I rolled back to use 1 single step could reduce the overhead of setting up the env, in this case, the `name` of each action should be explicit and readable, so naming it short wouldn't be a good idea. Fyi, I took the reference from google's android project https://github.com/android/compose-samples/blob/master/.github/workflows/Crane.yaml 🙇

Reply:
workflow name is already verbs imo 🤔  👉  lint & unit test 🤔